docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -47,11 +47,17 @@ Define the data you want GX Cloud to access within Snowflake.
 
 2. Click the **New Data Source** tab and then select **Snowflake**.
 
-3. Enter a meaningful name for the Data Asset in the **Data Source name** field.
+3. Select one of the following options:
 
-4. Optional. To use a connection string to connect to a Data Source, click the **Use connection string** selector, enter a connection string, and then move to step 6. The connection string format is: `snowflake://<user_login_name>:<password>@<accountname>`.
+    - If you're connecting to an org-hosted Snowflake Data Asset for the first time, copy the code and see [Connect GX Cloud to Snowflake](../connect/connect_snowflake.md).
```

Comment:
Copy what code?  Isn't this one of the processes that is entirely in the pop-out drawer in GX Cloud?

Reply:
@Rachel-Reverie, the user is instructed to "copy the code in the code pane." So, they know where it is. Prior to this instruction, they are instructed "in GX Cloud, click **Data Assets** > **New Data Asset** > **Snowflake**" or "in GX Cloud, click **Data Assets** > **New Data Asset** > **PostgreSQL**. Again, if you follow the procedure, you'll see what is being referenced and know what to do. There is very little risk a user won't know what to do or where to do it if they follow the instructions. For your reference, here is the updated UI:

![image](https://github.com/great-expectations/great_expectations/assets/104205257/a5706717-d2fb-4fbc-a928-bc152e560f5b)